gadopt/doc_descriptions.txt

```diff
@@ -0,0 +1,134 @@
+- Structure of documentation
+-> Index page with introductory text and list of clickable links underneath, each
+   pointing to the associated G-ADOPT module's API documentation.
+-> Left-side panel with foldable entries. When all folded, it looks similar to the list
+   on the index page. When an entry is unfolded, it reveals its associated items from
+   the documented API.
+-> Each module's page describes the API of that specific module and begins with
+   introductory text to describe the module's purpose.
+
+- User guide
+
+Welcome to G-ADOPT's user guide! In this guide, you will learn how to interact with the
+G-ADOPT library in order to easily tackle numerical problems associated with mantle
+convection, glacial isostatic adjustment, and groundwater. G-ADOPT provides a
```

Comment:
Rather than spell these out, I would just say "Simulate Geoscientific Flows" or similar. This avoids the need to update documentation like this if we add or remove specific areas.